omod/pom.xml

```diff
@@ -57,6 +57,12 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
```

Comment:
this is needed. otherwise it gives a bean creation exception. since we using  this
 <bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">  
in webModuleApplicationContext.xml